menuinst/winshortcut.cpp

```diff
@@ -30,18 +32,24 @@
 
 static PyObject *CreateShortcut(PyObject *self, PyObject *args)
 {
-    Py_UNICODE *path; /* path and filename */
-    Py_UNICODE *description;
-    Py_UNICODE *filename;
+    PyObject *py_path; /* path and filename */
```

Comment:
Don't we need to free (as in `Py_DECREF `) these objects at the end?

Reply:
I tried in #164 but it crashes. According to @ngoldbaum, these are borrowed references so they don't need decref'ing.

Reply:
in the docs for PyArg_ParseTuple, under the `O` code:

> The C program thus receives the actual object that was passed. A new strong reference to the object is not created (i.e. its reference count is not increased).

`U` is the same as `O` except is does a PyUnicode_Check as well.